### PR TITLE
Move sort field configuration from appearance page to a separate page

### LINKED ISSUE #941: Put Appearance page sections into tabs
The Administration > Appearance page contains several distinct configuration areas, plus we're going to need a new one for selecting and cropping the site masthead image. To make working with these configurations easier, I suggest we reorganize these sections into a tabbed interface (in case it's relevant, note that we might to do this other Admin/Curator pages eventually):

administration_-_appearance___maps_of_africa__an_online_exhibit_-_blacklight

Here's how each tabbed section might look (Site masthead details will part of a new ticket):

tabs

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #941 

screen shot 2015-02-20 at 4 30 23 pm

screen shot 2015-02-20 at 4 43 47 pm

### CODE REVIEW

File: app/views/spotlight/shared/_curation_sidebar.html.erb [hunk 1]
```diff
@@ -6,14 +6,13 @@
...
-    <% if can? :update, @exhibit.blacklight_configuration %>
-    <li><%= link_to t(:'spotlight.curation.sidebar.metadata'), spotlight.exhibit_edit_metadata_path(@exhibit) %></li>
-    <% end %>
     <% if can? :tag, @exhibit %>
     <li><%= link_to t(:'spotlight.curation.sidebar.tags'), spotlight.exhibit_tags_path(@exhibit) %></li>
     <% end %>
     <% if can? :update, @exhibit.blacklight_configuration %>
+    <li><%= link_to t(:'spotlight.curation.sidebar.metadata'), spotlight.exhibit_edit_metadata_path(@exhibit) %></li>
     <li><%= link_to t(:'spotlight.curation.sidebar.search_facets'), spotlight.exhibit_edit_facets_path(@exhibit) %></li>
+    <li><%= link_to t(:'spotlight.curation.sidebar.sort_fields'), spotlight.exhibit_edit_sort_fields_path(@exhibit) %></li>
```
Comment (jkeck) [inline app/views/spotlight/shared/_curation_sidebar.html.erb]: I know these existed before but should all of these `@exhibit`s be `current_exhibit`?

For some reason I thought we got these in this file already....

File: spec/controllers/spotlight/blacklight_configurations_controller_spec.rb [hunk 2]
```diff
@@ -75,6 +75,12 @@
       end
     end
 
+    describe "#edit_sort_fields" do
+      it "should be successful" do
+
+      end
```
Comment (jkeck) [inline spec/controllers/spotlight/blacklight_configurations_controller_spec.rb]: Cruft or should there be a test here?

**Comment by jkeck** [comment]
:+1: looks good, just a few minor comments.

File: spec/controllers/spotlight/blacklight_configurations_controller_spec.rb (hunk 2 above)
Comment (cbeer) [inline spec/controllers/spotlight/blacklight_configurations_controller_spec.rb]: Dunno what happened to it. I'll re-add it.

File: app/views/spotlight/shared/_curation_sidebar.html.erb (hunk 1 above)
Comment (cbeer) [inline app/views/spotlight/shared/_curation_sidebar.html.erb]: Good idea. I'll take care of it in a separate commit.

**Pull request merged**